docs/website/docs/guides/ml-frameworks/index.md

```diff
@@ -3,6 +3,18 @@
 IREE supports popular machine learning frameworks using the same underlying
 technology.
 
+``` mermaid
+graph LR
+  A[ML frameworks]
+  B[Imported MLIR]
+  C[IREE compiler]
+  D[Runtime deployment]
+
+  A --> B
+  B --> C
+  C --> D
+```
```

Comment:
If the general approach looks good, I can add accessibility labels and descriptions to each of these diagrams: https://mermaid.js.org/config/accessibility.html#acctitle-and-accdescr-usage-examples